### PR TITLE
fix(core): tabbing for lists, lists in cards

### LINKED ISSUE #8303: [Accessibility] List Card items are accessible via TAB key instead of arrow up/down keys only
Is this a bug, enhancement, or feature request?
--
A bug

Briefly describe your proposal
--
List items should be accessible via arrow up/down keys only

Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
--
Issue observed with Angular 13 + F-NGX 0.35.1

If this is a bug, please provide steps for reproducing it.
--
Focus onto the very first list item under this component, and then press TAB key will move on to the next list item

Screen Shot 2022-06-30 at 12 31 57 PM

Is there anything else we should know?
-- 
Related SAP Ariba JIRA defect: https://product-jira.ariba.com/browse/SS-36895

### PR DESCRIPTION
**Pull request by mikerodonnell89**
fixes #8303

### CODE REVIEW

File: libs/core/src/lib/card/card.component.ts [hunk 1]
```diff
@@ -83,6 +91,11 @@ export class CardComponent implements OnChanges, OnInit, CssClassBuilder, OnDest
         this.buildComponentCssClass();
     }
 
+    /** @hidden */
+    ngAfterContentInit(): void {
+        this.listItems?.first?.setIsFirst(true);
```
Comment (platon-rov) [inline libs/core/src/lib/card/card.component.ts:96]: the list can be added later (after content init), so it won't be caught. probably we have to manage the list item's tabindex in the list item code.

Comment (mikerodonnell89) [reply libs/core/src/lib/card/card.component.ts:96]: `the list can be added later (after content init), so it won't be caught`

Confused what you mean here, the card won't necessarily have a list so we need the ? check

`probably we have to manage the list item's tabindex in the list item code.`

That is what's happening in the `setIsFirst` function

Comment (g-cheishvili) [reply libs/core/src/lib/card/card.component.ts:96]: He meant that ngAfterContentInit is triggered only once. What if the component did not have a list in it during content init, but after some conditions were met in the client's app and the list appeared? In that case, this `setIsFirst` will not get triggered

**Review by g-cheishvili — CHANGES_REQUESTED** [review]
What Platon said

**Comment by mikerodonnell89** [comment]
Misunderstood the original linked issue so I'm closing this pr

**Pull request closed without merging**